### PR TITLE
Normalise

### PR DESCRIPTION
**Pull request by pietgeursen**

### CODE REVIEW

**Comment by pietgeursen** [comment]
@mixmix Reckon it's worth casting a careful eye over the schema in `./schema` to see if that's the right way to constrain the normalised objects.

File: test/index.test.js [hunk 1]
```diff
@@ -1,6 +1,58 @@
...
+      type: CHOOSE_ONE,
+      choice: 0
+    }
+  }
+  t.ok(isPosition(validPosition))
+
+  var parsedPosition = parsePosition(validPosition)
+  t.ok(isNormalisedPosition(parsedPosition))
+  t.end()
+})
```
Comment (mixmix) [inline test/index.test.js]: the interesting thing I want to test is : where is the case where something is v1 format, but doesn't produce a normalized poll.... 

How do we test that ... or do we always assert that it has been normalised after we parse?

File: schema/poll.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+const ssbSchemaDefintions = require('../v1/lib/ssbSchemaDefintions')
+
+const schema = {
+  $schema: 'http://json-schema.org/schema#',
+  type: 'object',
+  required: ['type', 'details', 'title', 'closesAt', 'version'],
```
Comment (mixmix) [inline schema/poll.js]: cosmetic :kiss: I like properties listed in the order they were mentioned in the required property. I think it's easier to read. superficial though

File: v1/index.js [hunk 3]
```diff
@@ -55,10 +55,10 @@ module.exports = {
       return isV1ChooseOnePoll(poll) ? true : undefined
     }
 
-    function parsePoll ({poll}) {
+    function parsePoll (poll) {
       if (!isV1Poll(poll)) { return }
 
-      return Poll({poll})
+      return Poll(poll)
```
Comment (mixmix) [inline v1/index.js]: should this be : 

```js
var parsed = Poll(poll)
if (!isNormalisedPoll(parsed)) throw new Error('NOPE')
return parsed
```

Comment (mixmix) [reply v1/index.js]: or could do it inside the parsePoll ...

**Pull request merged**